pkg/build/controller/image_change_controller_test.go

```diff
@@ -1,427 +0,0 @@
-package controller
-
-import (
-	"errors"
-	"fmt"
-	"strings"
-	"testing"
-
-	kerrors "k8s.io/apimachinery/pkg/api/errors"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	apirequest "k8s.io/apiserver/pkg/endpoints/request"
-	kapi "k8s.io/kubernetes/pkg/api"
-	"k8s.io/kubernetes/pkg/client/clientset_generated/internalclientset/fake"
-
-	buildapi "github.com/openshift/origin/pkg/build/api"
-	buildtest "github.com/openshift/origin/pkg/build/controller/test"
-	buildgenerator "github.com/openshift/origin/pkg/build/generator"
-	"github.com/openshift/origin/pkg/cmd/server/bootstrappolicy"
-	imageapi "github.com/openshift/origin/pkg/image/api"
-)
-
-func TestNewImageID(t *testing.T) {
```

Comment:
OK I've done the build generator related port.  And `make update` ran clean for me locally.  Minus the previously mentioned caveats, all analogous tests have been ported.

Updates pushed.